WcaOnRails/db/seeds/development/competitions.seeds.rb

```diff
@@ -128,7 +128,7 @@
         competition: future_competition,
         name: Faker::Name.name,
         personId: user.wca_id,
-        countryId: Faker::Address.country,
+        countryId: Faker::Address.country.slice(0, 50), # Ensure that the length of an address won't cause database error.
```

Comment:
Actually it looks weird, but while I've been seeding the database, it has raised an error and I was no idea what was it about. What's funnier the randomly generated countryId's length was 1 character more than the maximum.